### PR TITLE
Update: Remove dao dependency from txn table

### PR DESCRIPTION
**Pull request by rmeena840**

### CODE REVIEW

**Comment by rmeena840** [comment]
@robert-zaremba Please review it.

File: examples/dao/setup_sigma_dao.sql [hunk 1]
```diff
@@ -36,36 +36,31 @@ DECLARE
...
...4yByIoYg0QMgciKGINIiliJwZkDxAiKWIiJwRiDRAjEhBAAAEAJTUANAAiEiInCWIjEhBEIicFYiISQABZIicFYiUSQAAXIicFYiEEEkAAAQAyBCISRCInCSJmIkMyBCUSMwEQJBIQMwETMgMSEDMBACInCmISEDMBFCInC2ISEDMBEiInDGISEDMBESInEmISEERC/8AyBCUSMwEQIhIQMwEAIicKYhIQMwEHIicLYhIQMwEIIicMYhIQREL/lyI1AEL/YyEENQBC/1wkNQBC/1YjIycNNhwBUGM1CTUIMgQiEiInEWIyBw4QMgciKGIOECMqYiMNEEQ0CSMSQAB3NAgiJwdiE0AAXgA3ABoBKRJAAEc3ABoBJwQSQAAuNwAaAScOEkAAAQAiJw4iJw5iIypiCGYjJxNiIihiDkAAAiJDIycTIihiZkL/9CInBCInBGIjKmIIZkL/2iIpIiliIypiCGZC/80jJw02HAFQIicHYmZC/5QjJw02HAFQIicHYmZC/4UyBCINIjggMgMSECI4ECQSECI4EScIZBIQIjgUMgoSECI4EiMPEEQjKiMqYjMBEghmIkMjJwViIxJEMgQiDSI4IDIDEhAiOBAkEhAiOBEnCGQSECI4FDIKEhAiOBIjDxBEMwEgMgMSMwESKmQSEEQjJxQ2GgFmIycPNhoCZiMnFTYaA2Y2GgSAABJAANYjJxA2GgRmNhoFFzUCNhoGFzUDNhoHFzUENhoIFzUFNAIyBw1EIycRNAJmNAM0Ag0nF2Q0AzQCCQ4QJxhkNAM0AgkPEEQjKDQDZjQENAMNRCMrNARmNAUiEjQFJRIRNAUhBBIRRCMnBTQFZiMnBWIiEkAASSMnBWIlEkAAHyMnBWIhBBJAAAEAIycWNhoJZiMnBzEXZiMnCSNmIkMjJwo2GglmIycSNhoKF2YjJws2GgtmIycMNhoMF2ZC/9IjJwo2GglmIycLNhoKZiMnDDYaCxdmQv+5IycQgAZzaGEyNTZmQv8iMgQiEjEgMgMSEDYwACcIZBIQRLEkshAnCGSyETIKshQjshIjsgGzIkMyBCISRCJDI0M2GgIXIw02GgIXNhoDFwwQNhoGF3EANQc1BjQHEEQqNhoAF2cnBjYaARdnJxc2GgIXZycYNhoDF2cnDzYaBGeACGRhb19uYW1lNhoFZycINhoGF2ciQw==';
```
Comment (robert-zaremba) [inline examples/dao/setup_sigma_dao.sql:39]: why this has changed?

Comment (rmeena840) [reply examples/dao/setup_sigma_dao.sql:39]: SC was updated recently.

**Comment by rmeena840** [comment]
@robert-zaremba Can you approve it?

**Review by robert-zaremba — APPROVED** [review]

**Pull request merged**